command/job_tag.go

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/posener/complete"
+)
+
+type JobTagCommand struct {
+	Meta
+}
+
+func (c *JobTagCommand) Help() string {
+	helpText := `
+Usage: nomad job tag apply [options] <jobname>
+
+  Save a job version to prevent it from being garbage-collected and allow it to
+  be diffed and reverted by name.
+  
+  Example usage:
+ 
+    nomad job tag apply -name "My Golden Version" -description "The version of the job we can roll back to in the future if needed" <jobname>
```

Comment:
Wrap command line help text at 80 cols if you can. For an example I'd add a `\` to move the description into the line below (or just shorten the text)